src/main/java/club/sk1er/hytilities/handlers/chat/modules/blockers/GameIsStartingInBlocker.java

```diff
@@ -0,0 +1,30 @@
+package club.sk1er.hytilities.handlers.chat.modules.blockers;
+
+import club.sk1er.hytilities.config.HytilitiesConfig;
+import club.sk1er.hytilities.handlers.chat.ChatModule;
+import net.minecraftforge.client.event.ClientChatReceivedEvent;
+
+import java.util.regex.Pattern;
+
+public class GameIsStartingInBlocker extends ChatModule {
+
+    @Override
+    public int getPriority() {
+        return -2;
+    }
+
+    //                                                           it's always seconds, even with 1 second
```

Comment:
whats up with this comment lol

Reply:
i noticed it aligned perfectly with the string

Reply:
ok well dont